common/templates/uploadProgress.modal.html

```diff
@@ -0,0 +1,77 @@
+<div class="modal-header">
+	<button class="btn btn-default pull-right modal-close" type="button" ng-click="ctrl.cancel()">X</button>
+    <h3 class="modal-title">{{ctrl.title}}</h3>
+</div>
+<div class="modal-body">
+	<div class="row no-row-margin" ng-show="!ctrl.isCreateUploadJob && !ctrl.isFileExists && !ctrl.isUpload ">
+    	<div class="col-md-12 pad0-left pad0-right text-right">
+	    	<strong>Completed: {{ctrl.checksumCompleted}}/{{ctrl.noOfFiles}}</strong>
+    	</div>
+	    <div class="progress">
+	        <div class="progress-bar" role="progressbar" ng-style="{ 'width': ctrl.checksumProgress + '%' }" style="width: 0%;background-color: #8cacc7 !important;"></div>
```

Comment:
I'm seeing `style="width: 0%;background-color: #8cacc7 !important;"` repeated quite a few times. This should be a css class instead.